packages/teams-js/src/public/appWindow.ts

```diff
@@ -27,14 +33,17 @@ export interface IAppWindow {
   addEventListener(type: string, listener: Function): void;
 }
 
+/**
+ * Child app window object that can be used to send messages to and fro from parent window.
```

Comment:
>Child app window object [](http://example.com/codeflow?start=3&length=23)

Can you add more details about in what context a developer would use this type?

Reply:
Do you mean teamsjs frame context or in general how developers can use Child app window? like 
```
Developers might use child app window objects to create custom dialogs, settings pages, or other UI elements.

Reply:
Does this sound good?
```
 Child app windows can be used to create custom dialogs, settings pages, or other UI elements that are displayed within the hosts interface. Developers can create custom UI elements that are tailored to the app's specific needs and that seamlessly integrate with the rest of the hosts interface.

Reply:
Is this type actually used for any of those things besides dialogs?

Reply:
Based on Trevor's comment below it should be something like this
``` 
This an object your app can use to communicate with the Child window it opened that contains that task